docs/api-reference/mesh-layers/simple-mesh-layer.md

```diff
@@ -119,6 +119,14 @@ Whether to render the mesh in wireframe mode.
 This is an object that contains material props for [lighting effect](/docs/api-reference/core/lighting-effect.md) applied on extruded polygons.
 Check [the lighting guide](/docs/developer-guide/using-lighting.md#constructing-a-material-instance) for configurable settings.
 
+##### `_useMeshColors` (Boolean, optional)
+
+- Default: `false`
+
+Whether to color pixels using vertex colors supplied in the mesh (the `COLOR_0` or `colors` attribute). If vertex colors are provided they will be mixed (`multiplied`) with the instance colors, in the same way the `ScenegraphLayer` works. A white instance color will result in the unmodified vertex color being used.
```

Comment:
<del> the instance colors</del> the color supplied by `getColor`

<del>A white instance color will result in the unmodified vertex color being used.</del> This is explaining how to use `getColor` rather than `_useMeshColors`. This section should explain what `_useMeshColors: true` and `_useMeshColors: false` does, respectively.

Reply:
Updated.  

PS - That's some fancy use of  the `<del>` tag there...

Reply:
PS you can do the same with tildes `~strikethrough~`: ~strikethrough~